.env-dist

```diff
@@ -26,3 +26,5 @@ PGPASSWORD=""
 PGDATABASE="blurts"
 PGHOST="localhost"
 PGPORT=5432
+
+DATABASE_URL="postgres://postgres@localhost:5432/blurts"
```

Comment:
I like this single `DATABASE_URL` config, even if we don't use sequelize models.

Reply:
Should we just kill L24-28?

Reply:
When we remove the "raw" `pg` code we can remove this at the same time.